modules/installation-complete-user-infra.adoc

```diff
@@ -146,19 +146,22 @@ command.
 If the pod logs display, the Kubernetes API server can communicate with the
 cluster machines.
 
-ifndef::ibm-power[]
-. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing. Do not enable multipathing during installation.
+ifndef::ibm-power,ibm-z[]
+. For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing.
 +
 [NOTE]
 ====
 When installing with multipath, it is strongly recommended to enable it at installation time, and not at a later time, which can cause problems.
 ====
 +
 See "Enabling multipathing with kernel arguments on {op-system}" in the _Installing on bare metal_ documentation for more information.
-endif::ibm-power[]
+endif::ibm-power,ibm-z[]
 ifdef::ibm-power[]
+// Should the following sentence start with "For an installation with Fibre Channel Protocol (FCP)" as it does in the ibm-z ifdef?
 . Additional steps are required to enable multipathing. Do not enable multipathing during installation.
 +
+// Should the following sentence be changed to match ibm-z ifdef wording? For example:
+// See “Enabling multipathing with kernel arguments on {op-system}" in the _Post-installation machine configuration tasks documentation_ for more information.
 See the _Installing {op-system} and starting the {product-title} bootstrap process_ documentation for more information.
```

Comment:
@ktania46 or @SNiemann15 - Same question here, should this be updated to direct users to the post-install docs instead?